fianetsceau.php

```diff
@@ -471,9 +486,10 @@ public function sendXML($id_order)
 				else
 					$image_url = null;
 			}
-			$image_url = null;
-			$product_name = str_replace("'", '', $product['product_name']);
-			$product_name = str_replace('&', '', $product_name);
+			$image_url = null;	
+			
+			$product_name = str_replace(array('&', "'", '�'), array('', '', ''), $product['product_name']);

```

Comment:
Please have a look at the [module validator](https://validator.prestashop.com). Your file at the wrong encoding, and should be in utf-8